src/Artesian/_ClientsExecutor/ArtesianJsonSerializer.py

```diff
@@ -66,6 +66,19 @@ def __artesianDictDeserializer(
     return result
 
 
+def __artesianDictofListDeserializer(
+    obj: list, cls: type, *args: Any, **kwargs: Any
+) -> object:
+    result = []
+
+    for key in obj:
+        for valueD in key["Value"]:
+            result.append(jsons.load(key["Key"], str, *args, **kwargs) + ":" +
```

Comment:
this is not a Dict of List. is a list of strings  with a : separator.

can you please proof that the deserializer is able to parse the response correctly matching the structure aligned with typings? Typings are onky "hints" at compile time, at runtime you can have anything.

to do so, mock the json result using an hardcoded json string instead of the result of a serialization call and assert values into the expected structure.